production/grafana-agent-mixin/alerts.libsonnet

```diff
@@ -130,6 +130,16 @@ local _config = config._config;
               summary: '{{ $labels.job }} has used more than 10KB per series for more than 5 minutes',
             },
           },
+          {
+            alert: 'GrafanaAgentContainerRestarts',
+            expr: |||
+              sum by (pod) (rate(kube_pod_container_status_restarts_total{namespace="agent-smoke-test"}[10m])) > 0
+            |||,
+            'for': '2m',
```

Comment:
I think we can drop the `for` so it fires immediately whenever there's an unexpected restart. WDYT?

Reply:
that sounds good :+1: